test/drivers/test_suites/syscall_exit_suite/finit_module_x.cpp

```diff
@@ -0,0 +1,60 @@
+#include "../../event_class/event_class.h"
+
+#if defined(__NR_finit_module)
+
+#if defined(__NR_close) && defined(__NR_open) && defined(__NR_read)
```

Comment:
why do we need this `ifdef`?

Reply:
Ops, the test was using some random bytes from urandom, then I changed it.